better-font-awesome.php

```diff
@@ -128,21 +128,21 @@ class Better_Font_Awesome_Plugin {
      *
      * @return  Better_Font_Awesome  The BFA object.
      */
-    public static function get_instance( $args = '' ) {
-        static $instance = null;
-        if ( null === $instance ) {
-            $instance = new static( $args );
-        }
+	public static function get_instance( $args = '' ) {
 
-        return $instance;
-    }
+		if ( ! self::$instance ) {
+			self::$instance = new static( $args );
```

Comment:
But idea of the singleton is that constructor isn't publicly accessible.. Probably I'm missing something.